README.md

```diff
@@ -33,70 +33,92 @@ instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
 ### Run the test suite from your machine
 
-> **You will need to be connected to the VPN to test against Integration or
-> Staging.**
+Read "[Environment variables](#environment-variables)" for an explanation of the environment variables used in the examples below.
 
-The tests require additional configuration to run successfully on a local
-machine.
+*Run Smokey against Production*:
 
 ```sh
-ENVIRONMENT=integration \
+ENVIRONMENT=production
```

Comment:
You're no longer setting this in the environment, but the app still needs to see it.
```suggestion
export ENVIRONMENT=production
```